apps/site/pages/components/molecules/rating.mdx

```diff
@@ -0,0 +1,122 @@
+---
+description: Rating displays a product rating based on other shoppers' feedback.
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/AggregateRating___a8300916874f5a11de7633b7cabc12e9.png
+---
+
+import { Tab, Tabs } from 'nextra-theme-docs'
+import PropsSection from 'site/components/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+import { Rating, Icon } from '@faststore/ui'
+import { Star } from '@faststore/components'
+
+<header>
+
+# Rating
+
+The `Rating` component displays the average rating of a product based on other shoppers' feedback and reviews. `Rating` supports icons from our components.
+
+</header>
+
+## Overview
+
+<Tabs items={['Example', 'Code']} defaultIndex="0">
+  <Tab>
+    <OverviewSection>
+      <Rating value="3.5" icon={<Star />} />
+    </OverviewSection>
+    <OverviewSection>
+      <Rating value="0" icon={<Star />} actionable/>
+    </OverviewSection>
+  </Tab>
+  <Tab>
+
+    ```tsx
+    <Rating value="3.5" icon={<Star />} />
+    ```
+
+    ```tsx
+    <Rating value="0" icon={<Star />} actionable/>
```

Comment:
The value prop must be a value of type number and you provide a string, the rest of the file has other examples of misuse.

```suggestion
      <Rating value={3.5} icon={<Star />} />
    </OverviewSection>
    <OverviewSection>
      <Rating value={0} icon={<Star />} actionable/>
    </OverviewSection>
  </Tab>
  <Tab>

    ```tsx
    <Rating value={3.5} icon={<Star />} />
    ```

    ```tsx
    <Rating value={0} icon={<Star />} actionable/>
```